lib/oban/notifiers/postgres.ex

```diff
@@ -22,6 +22,11 @@ if Code.ensure_loaded?(Postgrex) do
 
     The notifications system is built on PostgreSQL's `LISTEN/NOTIFY` functionality. Notifications
     are only delivered **after a transaction completes** and are de-duplicated before publishing.
+    This means that notifications sent during a transaction will not be sent if the transaction is
+    rolled back, providing consistency; this is the only notifer which provides that guarantee.
+    However, it is not as scalable as other notifiers because because each notification requires a
+    separate query.
```

Comment:
```suggestion
    However, it is not as scalable as other notifiers because because each notification requires a
    separate query and notifications can't exceed 8kb.
```